src/status_im/switcher/home_stack.cljs

```diff
@@ -12,52 +13,41 @@
 (defn load-stack? [stack-id]
   (case stack-id
     :communities-stack @bottom-tabs/load-communities-tab?
-    :chats-stack @bottom-tabs/load-chats-tab?
-    :browser-stack @bottom-tabs/load-browser-tab?
-    :wallet-stack @bottom-tabs/load-wallet-tab?))
+    :chats-stack       @bottom-tabs/load-chats-tab?
+    :browser-stack     @bottom-tabs/load-browser-tab?
+    :wallet-stack      @bottom-tabs/load-wallet-tab?))
 
 (defn stack-view [stack-id shared-values]
   (when (load-stack? stack-id)
     [:f>
      (fn []
-       [reanimated/view {:style (reanimated/apply-animations-to-style
-                                 {:opacity        (get shared-values (get constants/stacks-opacity-keywords stack-id))
-                                  :pointer-events (get shared-values (get constants/stacks-pointer-keywords stack-id))}
-                                 {:top              0
-                                  :bottom           (if platform/ios? 79 54)
-                                  :left             0
-                                  :right            0
-                                  :position         :absolute})}
+       [reanimated/view
+        {:style (reanimated/apply-animations-to-style
+                 {:opacity        (get shared-values (get constants/stacks-opacity-keywords stack-id))
+                  :pointer-events (get shared-values (get constants/stacks-pointer-keywords stack-id))}
+                 {:position :absolute
+                  :top      0
+                  :bottom   0
+                  :left     0
+                  :right    0})}
         (case stack-id
```

Comment:
we must use bottomTabs from RNN instead, not sure if this should be done in this PR

Reply:
Hi, we discussed this, RNN bottomTabs don't provide that much customization to replicate figma designs. [discussion ](https://discord.com/channels/@me/881872253460049961/978626833320935465)

Also, please check out this video from current implementation and bottom tabs performance. ([video](https://discord.com/channels/624205794384281629/626744384565674034/1026491377393086516))

(yes, RNN bottomTabs are a little more performant, maybe we can discuss this design team in mobile-ui meeting?)

Reply:
probably that discussion is not relevant as we changed switcher to shell, I will look into this. Thank you

Reply:
Adding for ref:
https://user-images.githubusercontent.com/17097240/193815245-26efc0da-ab31-417e-975f-d3d5c47de0bf.mp4

Reply:
i mean, you can't just test it on high-end phone with no data, we had issues before on old android phones, and with RNN implementation they are all gone, the difference is the DOM, in the current implementation it will be huge, but i don't have any data on how different it will be , also we lost native buttons, but ok, lets make it implemented and we'll see how we could improve it later

Reply:
> lets make it implemented and we'll see how we could improve it later

cool, thank you

Reply:
> we had issues before on old android phones

Yes, I tried on the old phone, communities screen is a little heavy and doesn't feel that nice. I will like if we can opt for native bottom tabs, at this stage instead of later.

I will see if I can find way around above problems, In the meantime Please let me know, if you have any suggestions on how we can make native bottom tabs work while replicating designs.

(Or can we request, designs to be changed?)
Like we still can use jump to feature, (Just lose, transparent bottom tab, and bottom tab custom design). wdyt?

Reply:
I think one thing, we should keep an eye on is keeping home screens simple, to avoid loading time.
(because communities is the screen, which feels most laggy currently)